help_to_heat/frontdoor/views.py

```diff
@@ -14,7 +14,9 @@
 from ..portal import email_handler
 from . import eligibility, interface, schemas
 from .eligibility import calculate_eligibility, eco4
-from .interface import DuplicateReferralChecker
+from .session_handlers.duplicate_referral_checker import (
+    DuplicateReferralChecker,
+)
```

Comment:
Is this a formatting thing for line length?

Reply:
yeah, very conservative but it insists